.github/workflows/labeler.yml

```diff
@@ -10,37 +10,38 @@ on:
 jobs:
   draft_PR:
     if: (github.event.pull_request.draft == true)
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     name: Add WIP label
     steps:
       - name: Add WIP Label
-        uses: buildsville/add-remove-label@v1
+        uses: buildsville/add-remove-label@6008d7bd99d3baeb7c04033584e68f8ec80b198b # v1
         with:
           token: ${{secrets.LABEL_MODIFIER_TOKEN_GITHUB}}
           label: "WIP ⏳"
           type: add
+
       - name: Remove Ready for Review Label 
-        uses: buildsville/add-remove-label@v1
-        if: contains(github.event.pull_request.labels.*.name, 'Ready for review ✅')
+        uses: buildsville/add-remove-label@6008d7bd99d3baeb7c04033584e68f8ec80b198b # v1
         with:
           token: ${{secrets.LABEL_MODIFIER_TOKEN_GITHUB}}
```

Comment:
Just for my understanding 
For what reason are we using here an extra token instead of the `GITHUB_TOKEN`?

Reply:
To be honest, when I created the CI I didn't think about this option. I will test with the `GITHUB_TOKEN` and if it works fine, I don't know if it has the same permission, then I would change it everywhere to make it more standard.